.github/workflows/tap-tests.yml

```diff
@@ -4,7 +4,10 @@ on: [push, pull_request]
 jobs:
   run-babelfish-tap-tests:
     env:
-      INSTALL_DIR: psql
```

Comment:
why do we use tap tests instead of JDBC test framework?

Reply:
We can't test dump/restore in JDBC framework.